### PR TITLE
Indicators expiration

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20989

## Description
Added the expiration field for the indicators where it's relevant (for indicators whose default expiration behavior is to never expire, there is no need to add the field). Updated the indicators `toVersion` field to `5.4.9` and created a duplicate indicator file with `"fromVersion": "5.5.0"` and the `expiration` field with the appropriate value in minutes.

## Required version of Demisto
5.5.0

## Does it break backward compatibility?
   - No

## Must have
- ~~[ ] Tests~~
- [ ] Documentation (with link to it)
- [ ] Code Review

## Additional changes
Updated the associated type for the `region` indicator field so that the name of the `IPv6 CIDR` indicator matches that of the indicator's `details` field.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [reputation-cidr_CHANGELOG.md](https://github.com/demisto/content/edit/indicators-expiration/Misc/reputation-cidr_CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F5389)
- [ ] [reputation-domain_CHANGELOG.md](https://github.com/demisto/content/edit/indicators-expiration/Misc/reputation-domain_CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F5389)
- [ ] [reputation-fqdn_CHANGELOG.md](https://github.com/demisto/content/edit/indicators-expiration/Misc/reputation-fqdn_CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F5389)
- [ ] [reputation-ip_5.5_CHANGELOG.md](https://github.com/demisto/content/edit/indicators-expiration/Misc/reputation-ip_5.5_CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F5389)
- [ ] [reputation-ip_CHANGELOG.md](https://github.com/demisto/content/edit/indicators-expiration/Misc/reputation-ip_CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F5389)
- [ ] [reputation-ipv6_CHANGELOG.md](https://github.com/demisto/content/edit/indicators-expiration/Misc/reputation-ipv6_CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F5389)
- [ ] [reputation-ipv6_cidr_CHANGELOG.md](https://github.com/demisto/content/edit/indicators-expiration/Misc/reputation-ipv6_cidr_CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F5389)
- [ ] [reputation-url_CHANGELOG.md](https://github.com/demisto/content/edit/indicators-expiration/Misc/reputation-url_CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F5389)

### CODE REVIEW

**Comment by avidan-H** [comment]
@kirbles19 please take a look at the changelogs linked in the PR description above.

**Comment by kirbles19** [comment]
@avidan-H Done with all changelogs.

File: Misc/reputation-domain_CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Updated versioning.
```
Comment (ronykoz) [inline Misc/reputation-domain_CHANGELOG.md]:
```suggestion
- 
```

File: Misc/reputation-fqdn_CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## [Unreleased]
-
+Updated versioning.
```
Comment (ronykoz) [inline Misc/reputation-fqdn_CHANGELOG.md]:
```suggestion
-
```

File: Misc/reputation-cidr_CHANGELOG.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
 ## [Unreleased]
--
+Added the **expiration** field, which defines when indicators of this type expire (in minutes). The default value is 10,080 (7 days).
```
Comment (ronykoz) [inline Misc/reputation-cidr_CHANGELOG.md]:
```suggestion
Added the **expiration** field, which defines when indicators of this type expire (in minutes). The default value is 10,080 (7 days). This field is available from Demisto v5.5.
```

File: Misc/reputation-ip_CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Updated versioning.
```
Comment (ronykoz) [inline Misc/reputation-ip_CHANGELOG.md]:
```suggestion
-
```

File: Misc/reputation-ipv6_CHANGELOG.md [hunk 5]
```diff
@@ -1,2 +1,2 @@
 ## [Unreleased]
--
+Added the **expiration** field, which defines when indicators of this type expire (in minutes). The default value is 10,080 (7 days).
```
Comment (ronykoz) [inline Misc/reputation-ipv6_CHANGELOG.md]:
```suggestion
Added the **expiration** field, which defines when indicators of this type expire (in minutes). The default value is 10,080 (7 days). This field is available from Demisto v5.5.
```

File: Misc/reputation-ipv6_cidr_CHANGELOG.md [hunk 6]
```diff
@@ -1,2 +1,3 @@
 ## [Unreleased]
--
+Added the **expiration** field, which defines when indicators of this type expire (in minutes). The default value is 10,080 (7 days).
```
Comment (ronykoz) [inline Misc/reputation-ipv6_cidr_CHANGELOG.md]:
```suggestion
Added the **expiration** field, which defines when indicators of this type expire (in minutes). The default value is 10,080 (7 days). This field is available from Demisto v5.5.
```

File: Misc/reputation-url_CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Updated versioning.
```
Comment (ronykoz) [inline Misc/reputation-url_CHANGELOG.md]:
```suggestion
-
```

**Pull request merged**